include/mujincontrollerclient/mujincontrollerclient.h

```diff
@@ -696,8 +696,14 @@ class MUJINCLIENT_API WebResource
     template<class T>
     inline T Get(const std::string& field, double timeout = 5.0) {
         rapidjson::Document pt(rapidjson::kObjectType);
-        GetWrap(pt, field, timeout);
-        return mujinjson_external::GetJsonValueByKey<T>(pt, field.c_str());
+        std::string path = field;
+        if(path[0]!='/') {
+            path = std::string("/")+field;
+        }
+        std::string fieldToConstraint = path.substr(1);
```

Comment:
@cielavenir looks pretty inefficient with creating all these strings on the heap, especially when rapidjson doesn't require std::string

Reply:
well, alternative ideas are:

1. use GetJsonValueByPath only when field contains "/"
2. add a new API named GetPath

Reply:
added GetPath now